examples/blink/Main.cpp

```diff
@@ -13,9 +19,31 @@ int main()
     // ConfigureDefaultClockNucleo767ZI();
 
     static main_::StmEventInfrastructure eventInfrastructure;
+
     static main_::NUCLEO ui;
     static services::DebugLed debugLed(ui.ledGreen);
 
+#if defined(STM32WBA) | defined(STM32WB)
+    services::GpioPinInverted ledRed(ui.ledRed);
+    services::GpioPinInverted ledBlue(ui.ledBlue);
+    ledRed.Set(false);
+    ledBlue.Set(false);
+
+    ui.buttonOne.Config(hal::PinConfigType::input);
+    ui.buttonTwo.EnableInterrupt([&ledRed]()
+        {
+            ledRed.GetOutputLatch() == true ? ledRed.Set(false) : ledRed.Set(true);
+        },
+        hal::InterruptTrigger::fallingEdge);
+
+    ui.buttonTwo.Config(hal::PinConfigType::input);
+    ui.buttonOne.EnableInterrupt([&ledBlue]()
+        {
+            ledBlue.GetOutputLatch() == true ? ledBlue.Set(false) : ledBlue.Set(true);
+        },
+        hal::InterruptTrigger::risingEdge);
+#endif
+
```

Comment:
The "blink" example only needs to blink one LED, and should be as simple as possible

```suggestion
```

Reply:
Ok, removing it.